bower.json

```diff
@@ -20,7 +20,8 @@
   "homepage": "https://github.com/AqoviaElements/epi-cms-property",
   "ignore": [],  
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.4.0"
+    "polymer": "Polymer/polymer#^1.4.0",
+    "properties-from-ancestor-behavior": "~1.0"
```

Comment:
The change I recently made to `inventory-overview` actually moves the control over major.minor to source.

But nevertheless, after more discussing, I agree that for public components there will probably be more caution in the team about these numbers, so I'm changing it to a carrot ;)